includes/init_includes/init_sessions.php

```diff
@@ -147,7 +147,7 @@
  * verify the browser user agent if the feature is enabled
  */
 if (SESSION_CHECK_USER_AGENT === 'True') {
-    $http_user_agent = $_SERVER['HTTP_USER_AGENT'];
+    $http_user_agent = $_SERVER['HTTP_USER_AGENT'] ?? '';
     if (empty($_SESSION['SESSION_USER_AGENT'])) {
```

Comment:
Should the next line also check that `$http_user_agent` is not empty, since that is likely now?

Reply:
I don't think so - setting `$_SESSION['SESSION_USER_AGENT']` to blank means they will match, so no session destroy.  I think it's ok as is.